share/spice/just_delete_me/just_delete_me.js

```diff
@@ -0,0 +1,67 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_just_delete_me = function(api_result) {
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('just_delete_me');
+        }
+        var script = $('[src*="/js/spice/just_delete_me/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/just_delete_me\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query).split(" ")[0].toLowerCase();
+
+        if (!decodedQuery || decodedQuery.length < 4) {
+            return Spice.failed('just_delete_me');
+        }
+
+        var exact = api_result.find(function(item) { return item.name.toLowerCase() === decodedQuery}) ||
+                    api_result.find(function(item) { return item.domains &&
+                        ((typeof(item.domains) === "string" && item.domains.toLowerCase() === decodedQuery) ||
+                         typeof(item.domains) === "object" && item.domains.some(function(domain) {return domain.toLowerCase() === decodedQuery}) )
+                    }),
+            api_result = exact ? [exact] : api_result.filter(function (item) {
+                        item.domains =  !item.domains ? [] : typeof(item.domains) === "string" ? [item.domains] : item.domains;
+                        if (item.name.toLowerCase().includes(decodedQuery) || item.domains.some(function(domain) {return domain.toLowerCase().includes(decodedQuery);}))
+                            return true;
+                        });
+
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('just_delete_me');
+        }
+        Spice.add({
+            id: "JustDeleteMe",
+            name: "Answer",
+            meta: {
+                sourceName: 'Just Delete Me',
+                sourceUrl: "http://justdelete.me/#" + decodedQuery
+            },
+            data: api_result,
+            normalize: function(item) {
+                return {
+                    delete_url: item.url,
+                    title: item.name,
+                    url: "http://justdelete.me/#" + decodedQuery,
+                    subtitle: "Difficulty: " + DDG.capitalize(item.difficulty),
+                    description: item.notes
+                };
+            },
+            relevancy: {
+                dup: 'name'
+            },
+            templates: {
+                group: 'text',
+                options: {
+                    footer: Spice.just_delete_me.footer,
+                    content: Spice.just_delete_me.content,
+                    moreAt: true,
+                    aux: false
+                },
+                variants: {
+                    tileTitle: '1line-large',
+                    tileFooter: '2line'
```

Comment:
I suggest we go with large, as the empty tiles will look just as empty as small but with large we won't have to tinker with the condensing of texts as it works correctly already without a custom style.
![screen shot 2016-02-25 at 20 05 04](https://cloud.githubusercontent.com/assets/1518025/13330866/2202dce0-dbfb-11e5-8e9e-75a7979812af.png)